setup.py

```diff
@@ -1,3 +1,12 @@
 from setuptools import setup
 
-setup()
+
+def local_scheme(version) -> str:
```

Comment:
add this change to avoid versioning like `1.38.11.dev13+g9aabe7c` which is invalid version name for PyPI
https://github.com/pypa/setuptools_scm/issues/342